pkg/controller/clone-controller_test.go

```diff
@@ -701,7 +709,7 @@ func createCloneReconciler(objects ...runtime.Object) *CloneReconciler {
 	cdiv1.AddToScheme(s)
 	rec := record.NewFakeRecorder(1)
 	// Create a fake client to mock API calls.
-	cl := fake.NewFakeClientWithScheme(s, objs...)
+	cl := fake.NewFakeClientWithScheme(s, objs...) //nolint:staticcheck // todo: BuildNameToCertificate() is deprecated - check this
```

Comment:
copy/paste error, comment should say `NewFakeClientWithScheme`

Reply:
Done